js/main.js

```diff
@@ -0,0 +1,134 @@
+var $todayOpen = document.querySelector('.today-open');
+var $PreviosClose = document.querySelector('.previos-close');
+var $displayTable = document.querySelector('.displayTable');
+var $view = document.querySelectorAll('.view');
+var $close = document.querySelector('#close');
+var searchListSymbol = [];
+var searchListName = [];
+var stockHigh = [];
+var stockLow = [];
+var stockOpen = [];
+var stockClose = [];
+var stockDateTime = [];
+$seachBtn.addEventListener('click', seachStock);
+$displayTable.addEventListener('click', displayTableClick);
+$close.addEventListener('click', closeSpecificStock);
+
+function closeSpecificStock() {
+  $seachBar.value = '';
+  switchView('home-view');
+}
+function seachStock(event) {
+  seachStockAPI($seachBar.value);
+  switchView('search-view');
+}
+
+function displayTableClick(event) {
+  if (!event.target.matches('I')) {
+    return;
+  }
+
+  getSpecificStockAPI(event.target.getAttribute('stock-symbol'));
+  switchView('stock-view');
+}
+
+function seachStockAPI(keyword) {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://www.alphavantage.co/query?function=SYMBOL_SEARCH&keywords=' + keyword + '&apikey=0GXU3KBC6YFBOGIB');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    for (var i in xhr.response.bestMatches) {
+      searchListSymbol.push(xhr.response.bestMatches[i]['1. symbol']);
+      searchListName.push(xhr.response.bestMatches[i]['2. name']);
+    }
+    showList();
+  });
+  xhr.send();
+}
+
+function showList() {
+  var $trNodes = document.querySelectorAll('tr');
+  for (var j = 0; j < $trNodes.length; j++) {
+    $trNodes[j].remove();
+  }
+  for (var i = 0; i < searchListName.length; i++) {
+    var $tr = document.createElement('tr');
+    var $tdName = document.createElement('td');
+    var $tdView = document.createElement('td');
+    var $i = document.createElement('i');
+    $tdName.setAttribute('class', 'width-90');
+    $tdView.setAttribute('class', 'width-10');
+    $i.setAttribute('class', 'fa fa-eye');
+    $i.setAttribute('stock-symbol', searchListSymbol[i]);
+    $tr.appendChild($tdName);
+    $tr.appendChild($tdView);
+    $tdView.appendChild($i);
+    $tdName.innerText = searchListName[i] + ' - ' + searchListSymbol[i];
+    $tbody.appendChild($tr);
+  }
+
+  searchListName = [];
+  searchListSymbol = [];
+}
+
+function getSpecificStockAPI(specificStock) {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://cloud.iexapis.com/stable/stock/' + specificStock + '/quote?token=sk_d5ca9aca3c0c446b93e9d5013e8d4a95');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    $stockName.innerText = xhr.response.companyName;
+    $PreviosClose.innerText = xhr.response.previousClose;
+    var change = '' + xhr.response.change;
+    $stockCurrentPrice.innerHTML = xhr.response.latestPrice + ' &nbsp&nbsp ' + xhr.response.change + ' (' + (parseFloat(xhr.response.changePercent) * 100).toFixed(2) + '%)';
```

Comment:
Instead of editing `innerHTML` directly, could you do this with a DOM creation function?